src/platform/device.gni

```diff
@@ -128,6 +125,14 @@ declare_args() {
   chip_device_config_enable_joint_fabric = false
 }
 
+declare_args() {
```

Comment:
why are we creating separate declare args? Please merge this and the joint fabric above as well to above line 121.

Reply:
I will merge them.